docs/troubleshooting/semgrep.md

```diff
@@ -48,21 +48,22 @@ semgrep:
 
 ### Other
 
-We currently don't have instructions for other providers, but logs for the action are always saved in `.semgrep_logs/`. There will be two files, `semgrep_agent_logs` and `semgrep_agent_output`. The former is a more verbose logging of what happened; the second contains the output that the action collected for semgrep. If you are running in docker, you can find the logs there.
-
-## Troubleshooting Semgrep CI
+Logs for the action are always saved in `.semgrep_logs/`. There are two files, `semgrep_agent_logs` and `semgrep_agent_output`. The former is a more verbose logging of what happened; the second contains the output that the action collected for semgrep. If you are running in docker, you can find the logs there.
```

Comment:
Hi there @brendongo @underyx! Just looking at this troubleshooting section  - "Logs for the action" - Does this mean GitHub action or semgrep-action? (Needs to be specified.)

How are the files `semgrep_agent_logs` and `semgrep_agent_output` named now?
`semgrep_ci_logs` and `semgrep_ci_output` or just `semgrep_logs` and `semgrep_output`?

I think that I should be able to test such things myself... so maybe train me a bit so I can try this myself.

Reply:
It means semgrep-agent. If you run `semgrep ci --config p/ci` on your local machine now, it should generate `~/.semgrep/last.log` and `~/.semgrep/semgrep_rules.yml`

Reply:
Ok @brendongo, including two subsections:

```
### Retrieving Semgrep CI logs

When you run `semgrep ci --config p/ci` logs are saved in `~/.semgrep/last.log`. Some more information can be found in `~/.semgrep/semgrep_rules.yml`.

#### Legacy Semgrep Action

There are two files, `semgrep_agent_logs` and `semgrep_agent_output`. The former is more verbose logging of what happened; the second contains the output that the action collected for semgrep. If you are running in Docker, you can find the logs there.
```